src/main/resources/application.properties

```
@@ -1 +1,11 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/rick_and_morty?serverTimezone=UTC
+spring.datasource.username=root
+spring.datasource.password=Romaxa051979
+#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect
```

Is it required property? if not should be removed
```suggestion
spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect
```